### PR TITLE
Helm chart

### PR DESCRIPTION
**Pull request by ilackarms**
adds helm chart for supergloo + value generation script

### CODE REVIEW

File: install/helm/supergloo/values-defaults.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+########################################################################
```
Comment (rickducott) [inline install/helm/supergloo/values-defaults.yaml]: Why call this values-defaults and not values-template.yaml?

Comment (ilackarms) [reply install/helm/supergloo/values-defaults.yaml]: it's not accurate to call it a template. the values in this file set defaults which the generate script overrides

File: install/helm/supergloo/Chart-template.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+appVersion: "0.1.0"
```
Comment (rickducott) [inline install/helm/supergloo/Chart-template.yaml]: Shouldn't appVersion be left out of the template? (and filled in by the release tag?)

File: install/helm/supergloo/generate-values.go [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	glooGenerate "github.com/solo-io/gloo/install/helm/gloo/generate"
+	"github.com/solo-io/solo-kit/pkg/utils/log"
+	"github.com/solo-io/supergloo/install/helm/supergloo/generate"
+)
+
+const (
+	valuesTemplate = "install/helm/supergloo/values-defaults.yaml"
```
Comment (rickducott) [inline install/helm/supergloo/generate-values.go]: Change variable name

File: install/helm/supergloo/values-defaults.yaml (hunk 1 above)
Comment (rickducott) [reply install/helm/supergloo/values-defaults.yaml]: I didn’t think there are overrides, at least in gloo — sounds good though

**Comment by ilackarms** [comment]
/kick

note: seems to be a flake: 

Step #3 - "test": SS
Step #3 - "test": ------------------------------
Step #3 - "test": /workspace/gopath/src/github.com/solo-io/supergloo/pkg/install/utils/helm/helm_chart_installer_test.go:95
Step #3 - "test": 
Step #3 - "test": not to have occurred
Step #3 - "test": Internal error occurred: resource quota evaluates timeout
Step #3 - "test": }
Step #3 - "test": },
Step #3 - "test": Code: 500,
Step #3 - "test": },
Step #3 - "test": RetryAfterSeconds: 0,
Step #3 - "test": ],
Step #3 - "test": },
Step #3 - "test": Field: "",
Step #3 - "test": Message: "resource quota evaluates timeout",
Step #3 - "test": Type: "",
Step #3 - "test": {
Step #3 - "test": Causes: [
Step #3 - "test": UID: "",
Step #3 - "test": Kind: "",
Step #3 - "test": Group: "",
Step #3 - "test": Name: "",
Step #3 - "test": Details: {
Step #3 - "test": Reason: "InternalError",
Step #3 - "test": Message: "Internal error occurred: resource quota evaluates timeout",
Step #3 - "test": Status: "Failure",
Step #3 - "test": ListMeta: {SelfLink: "", ResourceVersion: "", Continue: ""},
Step #3 - "test": TypeMeta: {Kind: "Status", APIVersion: "v1"},
Step #3 - "test": ErrStatus: {
Step #3 - "test": <*errors.StatusError | 0xc0009aea20>: {
Step #3 - "test": Expected error:
Step #3 - "test": 
Step #3 - "test": /workspace/gopath/src/github.com/solo-io/supergloo/pkg/install/utils/helm/helm_chart_installer_test.go:71

https://console.cloud.google.com/cloud-build/builds/9dda6836-f1b7-4a3d-a628-6e8747d64c5e?project=solo-public

**Pull request merged**